cogs/induct.py

```diff
@@ -356,7 +356,7 @@ async def induct(self, ctx: TeXBotApplicationContext, str_induct_member_id: str,
         await self._perform_induction(ctx, induct_member, silent=silent)
 
 
-class InductUserCommandsCog(BaseInductCog):
+class InductContextCommandsCog(BaseInductCog):
     """Cog class that defines the context menu induction commands & their call-back methods."""
```

Comment:
Term "context-menu" should be hyphenated.

Reply:
This issue was not introduced in this pr and as it is completely unrelated to the purpose of the pr are out of scope